app/helpers/search_helper.rb

```diff
@@ -63,14 +63,25 @@ def display_last_modified(record)
       DateTime.parse(revision_date).to_s(:date)
     end
   end
-  
-  # search dropdown is smaller if radio buttons are hidden in DMMT
-  def proposal_mode_short_dropdown
-     ' search-dropdown-short' if Rails.configuration.proposal_mode
+
+  def dropdown_size_modification
+    # search dropdown is smaller if radio buttons are hidden in DMMT
+    return ' search-dropdown-short' if Rails.configuration.proposal_mode
+
+    # TODO: Make this a permanent change when the UMM-T toggle is removed.
+    # search dropdown is wider if tools radio button is enabled
+    ' search-dropdown-wide' if Rails.configuration.umm_t_enabled
+  end
+
+  # TODO: Make this a permanent change when the UMM-T toggle is removed.
+  def search_field_size_modification
+    return if Rails.configuration.proposal_mode
+
+    'search-field-wide' if Rails.configuration.umm_t_enabled
```

Comment:
why does line 80 have no space at the beginning while line 73 and 69 do?

Reply:
This is a change from my PR, not his.  The answer is that the dropdown size modification is adding to a list of classes and the search field size modification is the only class.

Reply:
the function does not know how it is being used, should the space not be in the calling code so that these methods 'could' be reused for any condition?

Reply:
The goal is to get specific css applied, so I would argue that 80 is the one that should be changed (to have a space) rather than removing the spaces from the first function if you're trying to make these more reusable (though, by the nature of where this is used, I am skeptical we will find a place to reuse it).

Reply:
How should it be reusable? Should it be reusable to apply the css attached to those classes wherever this is called? If so, I think that 80 needs a space and 73 and 69 should remain the same. If the goal is to apply the css, I think you want the class added and not appended to some other class.  If it should be reusable such that the text is appendable to an existing class to create some new class that does not have any currently attached css, then 73 and 69 should drop their spaces.  I think the first case is more likely to be reused, but I expect that neither would be.

Reply:
I will make these more uniform in MMT-2267.  Discussed offline, all should have a space.